ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimSubmissionOperationIndicatorMapper.java

```diff
@@ -1,35 +1,40 @@
 package uk.gov.hmcts.cmc.ccd.mapper;
 
+import org.springframework.stereotype.Component;
 import uk.gov.hmcts.cmc.ccd.domain.CCDClaimSubmissionOperationIndicators;
 import uk.gov.hmcts.cmc.domain.models.ClaimSubmissionOperationIndicators;
 
-import java.util.function.Function;
+@Component
+public class ClaimSubmissionOperationIndicatorMapper {
```

Comment:
Can we please add unit tests for this as well? I know it's an exiting one, would be nice if you could add it's unit tests otherwise add this in tech debt.